### PR TITLE
add geolocation

### PR DESCRIPTION
**Pull request by seb879**
add geolocation to the map

### CODE REVIEW

File: src/location.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+navigator.geolocation.getCurrentPosition(function (position){
+    let coordonnees = []
+    coordonnees.push(position.coords.latitude)
+    coordonnees.push(position.coords.longitude)
+    return coordonnees
+});
```
Comment (erabin05) [inline src/location.js]: Why did you made this function to export if you never use it on the other files ?

Comment (erabin05) [reply src/location.js]: And I am not sure it works

**Review by erabin05 — CHANGES_REQUESTED** [review]
- Do you really need the file location.js ?

File: src/location.js (hunk 1 above)
Comment (seb879) [reply src/location.js]: I have delete this file

**Review by erabin05 — APPROVED** [review]

**Pull request merged**